portfolio/src/main/webapp/script.js

```diff
@@ -127,9 +115,15 @@ function expandDesign(clickedId){
                 This piece is meant to be worn as a shirt design and while the design was made with procreate, the 
                 text was done with illustrator.`
             ]
-        ]
-    }
+  ];
+ expandDesign(page_heading,page_heading_desc,img_ids_desc); 
+}
 
+function expandDesign(page_heading,page_heading_desc,img_ids_desc){
+    //whenever you click an image on the description page, this pulls up more images in the category alongside descriptions
+    const designPage = document.getElementById('design_container');
+    designPage.innerHTML = "";
+ 
     var button = document.createElement("button");
     var buttonText = document.createTextNode("Back");
     button.appendChild(buttonText);
```

Comment:
I find that defining fewer local variables makes code easier to read, because each local variable is one more thing for the reader to remember. A good rule of thumb is: Don't define a variable that's only used once.

Applying that rule here yields: `button.appendChild(document.createTextNode("Back"))`